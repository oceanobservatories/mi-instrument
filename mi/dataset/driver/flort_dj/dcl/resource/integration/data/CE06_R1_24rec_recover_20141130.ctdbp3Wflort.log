2014/11/30 00:28:11.959 [ctdbp3:DLOGP6]:Logger started: Idle state, without initialize
2014/11/30 00:30:14.979 [ctdbp3:DLOGP6]:Instrument Started [Power On]
2014/11/30 00:30:21.911 [ctdbp3:DLOGP6]: 11.8759,  3.39615,    1.136, 0, 0, 0, 30 Nov 2014 00:30:17
2014/11/30 00:31:03.917 [ctdbp3:DLOGP6]:Instrument Stopped [Power Off]
2014/11/30 01:28:10.296 [ctdbp3:DLOGP6]:Logger started: Idle state, without initialize
2014/11/30 01:30:22.387 [ctdbp3:DLOGP6]:Instrument Started [Power On]
2014/11/30 01:30:29.326 [ctdbp3:DLOGP6]: 11.7871,  3.38670,    1.096, 1926, 124, 88, 30 Nov 2014 01:30:24
2014/11/30 01:31:05.336 [ctdbp3:DLOGP6]:Instrument Stopped [Power Off]
2014/11/30 02:28:06.823 [ctdbp3:DLOGP6]:Logger started: Idle state, without initialize
2014/11/30 02:30:15.882 [ctdbp3:DLOGP6]:Instrument Started [Power On]
2014/11/30 02:30:22.810 [ctdbp3:DLOGP6]: 11.7010,  3.37536,    1.224, 0, 0, 0, 30 Nov 2014 02:30:18
2014/11/30 02:31:04.817 [ctdbp3:DLOGP6]:Instrument Stopped [Power Off]
2014/11/30 03:28:09.748 [ctdbp3:DLOGP6]:Logger started: Idle state, without initialize
2014/11/30 03:30:22.978 [ctdbp3:DLOGP6]:Instrument Started [Power On]
2014/11/30 03:30:29.918 [ctdbp3:DLOGP6]: 11.6688,  3.37494,    1.047, 1902, 125, 87, 30 Nov 2014 03:30:25
2014/11/30 03:31:03.927 [ctdbp3:DLOGP6]:Instrument Stopped [Power Off]
2014/11/30 04:28:09.306 [ctdbp3:DLOGP6]:Logger started: Idle state, without initialize
2014/11/30 04:30:14.341 [ctdbp3:DLOGP6]:Instrument Started [Power On]
2014/11/30 04:30:21.272 [ctdbp3:DLOGP6]: 11.5903,  3.36872,    1.304, 0, 0, 0, 30 Nov 2014 04:30:16
2014/11/30 04:31:03.275 [ctdbp3:DLOGP6]:Instrument Stopped [Power Off]
2014/11/30 05:28:09.117 [ctdbp3:DLOGP6]:Logger started: Idle state, without initialize
2014/11/30 05:30:22.516 [ctdbp3:DLOGP6]:Instrument Started [Power On]
2014/11/30 05:30:29.453 [ctdbp3:DLOGP6]: 11.5464,  3.36545,    1.050, 1782, 125, 86, 30 Nov 2014 05:30:24
2014/11/30 05:31:05.462 [ctdbp3:DLOGP6]:Instrument Stopped [Power Off]
2014/11/30 06:28:09.108 [ctdbp3:DLOGP6]:Logger started: Idle state, without initialize
2014/11/30 06:30:16.128 [ctdbp3:DLOGP6]:Instrument Started [Power On]
2014/11/30 06:30:23.057 [ctdbp3:DLOGP6]: 11.4899,  3.36079,    1.632, 0, 0, 0, 30 Nov 2014 06:30:18
2014/11/30 06:31:05.061 [ctdbp3:DLOGP6]:Instrument Stopped [Power Off]
2014/11/30 07:28:09.933 [ctdbp3:DLOGP6]:Logger started: Idle state, without initialize
2014/11/30 07:30:16.982 [ctdbp3:DLOGP6]:Instrument Started [Power On]
2014/11/30 07:30:23.920 [ctdbp3:DLOGP6]: 11.4895,  3.33245,    1.265, 0, 0, 0, 30 Nov 2014 07:30:19
2014/11/30 07:31:03.929 [ctdbp3:DLOGP6]:Instrument Stopped [Power Off]
2014/11/30 08:28:09.647 [ctdbp3:DLOGP6]:Logger started: Idle state, without initialize
2014/11/30 08:30:16.546 [ctdbp3:DLOGP6]:Instrument Started [Power On]
2014/11/30 08:30:23.476 [ctdbp3:DLOGP6]: 11.3430,  3.30890,    1.357, 0, 0, 0, 30 Nov 2014 08:30:18
2014/11/30 08:31:05.543 [ctdbp3:DLOGP6]:Instrument Stopped [Power Off]
2014/11/30 09:28:12.354 [ctdbp3:DLOGP6]:Logger started: Idle state, without initialize
2014/11/30 09:30:15.334 [ctdbp3:DLOGP6]:Instrument Started [Power On]
2014/11/30 09:30:22.264 [ctdbp3:DLOGP6]: 11.2950,  3.29986,    1.618, 0, 0, 0, 30 Nov 2014 09:30:17
2014/11/30 09:31:04.282 [ctdbp3:DLOGP6]:Instrument Stopped [Power Off]
2014/11/30 10:28:09.297 [ctdbp3:DLOGP6]:Logger started: Idle state, without initialize
2014/11/30 10:30:23.528 [ctdbp3:DLOGP6]:Instrument Started [Power On]
2014/11/30 10:30:29.497 # 11.1403,  3.29082,    1.138, 1036, 126, 86, 30 Nov 2014 10:30:24
2014/11/30 10:31:06.527 [ctdbp3:DLOGP6]:Instrument Stopped [Power Off]
2014/11/30 11:28:07.188 [ctdbp3:DLOGP6]:Logger started: Idle state, without initialize
2014/11/30 11:30:20.648 [ctdbp3:DLOGP6]:Instrument Started [Power On]
2014/11/30 11:30:27.594 [ctdbp3:DLOGP6]: 11.2627,  3.32095,    1.052, 1273, 121, 89, 30 Nov 2014 11:30:22
2014/11/30 11:31:05.606 [ctdbp3:DLOGP6]:Instrument Stopped [Power Off]
2014/11/30 12:28:10.239 [ctdbp3:DLOGP6]:Logger started: Idle state, without initialize
2014/11/30 12:30:15.142 [ctdbp3:DLOGP6]:Instrument Started [Power On]
2014/11/30 12:30:22.074 [ctdbp3:DLOGP6]: 11.2116,  3.32285,    3.848, 0, 0, 0, 30 Nov 2014 12:30:17
2014/11/30 12:31:04.136 [ctdbp3:DLOGP6]:Instrument Stopped [Power Off]
2014/11/30 13:28:08.626 [ctdbp3:DLOGP6]:Logger started: Idle state, without initialize
2014/11/30 13:30:15.508 [ctdbp3:DLOGP6]:Instrument Started [Power On]
2014/11/30 13:30:22.441 [ctdbp3:DLOGP6]: 11.1723,  3.31861,    2.241, 0, 0, 0, 30 Nov 2014 13:30:17
2014/11/30 13:31:04.456 [ctdbp3:DLOGP6]:Instrument Stopped [Power Off]
2014/11/30 14:28:10.449 [ctdbp3:DLOGP6]:Logger started: Idle state, without initialize
2014/11/30 14:30:15.332 [ctdbp3:DLOGP6]:Instrument Started [Power On]
2014/11/30 14:30:22.262 [ctdbp3:DLOGP6]: 11.1013,  3.29633,    2.397, 0, 0, 0, 30 Nov 2014 14:30:17
2014/11/30 14:31:04.280 [ctdbp3:DLOGP6]:Instrument Stopped [Power Off]
2014/11/30 15:28:06.714 [ctdbp3:DLOGP6]:Logger started: Idle state, without initialize
2014/11/30 15:30:15.568 [ctdbp3:DLOGP6]:Instrument Started [Power On]
2014/11/30 15:30:22.486 [ctdbp3:DLOGP6]: 11.0341,  3.28592,    2.398, 0, 0, 0, 30 Nov 2014 15:30:17
2014/11/30 15:31:04.505 [ctdbp3:DLOGP6]:Instrument Stopped [Power Off]
2014/11/30 16:28:07.660 [ctdbp3:DLOGP6]:Logger started: Idle state, without initialize
2014/11/30 16:30:18.991 [ctdbp3:DLOGP6]:Instrument Started [Power On]
2014/11/30 16:30:25.939 [ctdbp3:DLOGP6]: 10.9082,  3.27400,    1.086, 977, 125, 85, 30 Nov 2014 16:30:21
2014/11/30 16:31:03.971 [ctdbp3:DLOGP6]:Instrument Stopped [Power Off]
2014/11/30 17:28:10.405 [ctdbp3:DLOGP6]:Logger started: Idle state, without initialize
2014/11/30 17:30:17.249 [ctdbp3:DLOGP6]:Instrument Started [Power On]
2014/11/30 17:30:24.179 [ctdbp3:DLOGP6]: 10.8634,  3.27407,   47.401, 0, 0, 0, 30 Nov 2014 17:30:19
2014/11/30 17:31:04.197 [ctdbp3:DLOGP6]:Instrument Stopped [Power Off]
2014/11/30 18:28:10.443 [ctdbp3:DLOGP6]:Logger started: Idle state, without initialize
2014/11/30 18:30:17.277 [ctdbp3:DLOGP6]:Instrument Started [Power On]
2014/11/30 18:30:24.208 [ctdbp3:DLOGP6]: 10.8888,  3.27275,   53.921, 0, 0, 0, 30 Nov 2014 18:30:19
2014/11/30 18:31:04.246 [ctdbp3:DLOGP6]:Instrument Stopped [Power Off]
2014/11/30 19:28:08.734 [ctdbp3:DLOGP6]:Logger started: Idle state, without initialize
2014/11/30 19:30:15.584 [ctdbp3:DLOGP6]:Instrument Started [Power On]
2014/11/30 19:30:22.514 [ctdbp3:DLOGP6]: 10.8395,  3.25213,   54.157, 0, 0, 0, 30 Nov 2014 19:30:17
2014/11/30 19:31:04.542 [ctdbp3:DLOGP6]:Instrument Stopped [Power Off]
2014/11/30 20:28:07.642 [ctdbp3:DLOGP6]:Logger started: Idle state, without initialize
2014/11/30 20:30:16.456 [ctdbp3:DLOGP6]:Instrument Started [Power On]
2014/11/30 20:30:23.389 [ctdbp3:DLOGP6]: 10.9188,  3.25238,   15.311, 0, 0, 0, 30 Nov 2014 20:30:18
2014/11/30 20:31:03.406 [ctdbp3:DLOGP6]:Instrument Stopped [Power Off]
2014/11/30 21:28:09.858 [ctdbp3:DLOGP6]:Logger started: Idle state, without initialize
2014/11/30 21:30:14.783 [ctdbp3:DLOGP6]:Instrument Started [Power On]
2014/11/30 21:30:21.715 [ctdbp3:DLOGP6]: 10.9795,  3.25184,   55.247, 0, 0, 0, 30 Nov 2014 21:30:17
2014/11/30 21:31:05.772 [ctdbp3:DLOGP6]:Instrument Stopped [Power Off]
2014/11/30 22:28:10.054 [ctdbp3:DLOGP6]:Logger started: Idle state, without initialize
2014/11/30 22:30:16.948 [ctdbp3:DLOGP6]:Instrument Started [Power On]
2014/11/30 22:30:23.880 [ctdbp3:DLOGP6]: 10.8724,  3.24866,   40.661, 0, 0, 0, 30 Nov 2014 22:30:19
2014/11/30 22:31:03.908 [ctdbp3:DLOGP6]:Instrument Stopped [Power Off]
2014/11/30 23:28:10.423 [ctdbp3:DLOGP6]:Logger started: Idle state, without initialize
2014/11/30 23:30:15.217 [ctdbp3:DLOGP6]:Instrument Started [Power On]
2014/11/30 23:30:22.167 [ctdbp3:DLOGP6]: 10.7593,  3.24225,   15.275, 0, 0, 0, 30 Nov 2014 23:30:17
2014/11/30 23:31:04.246 [ctdbp3:DLOGP6]:Instrument Stopped [Power Off]
