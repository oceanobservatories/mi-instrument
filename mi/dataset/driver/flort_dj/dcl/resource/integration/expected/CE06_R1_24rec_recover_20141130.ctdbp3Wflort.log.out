
 subsite  | node  | sensor       | bin        | method    | time                  | deployment | internal_timestamp | port_timestamp        | preferred_timestamp | quality_flag | date_string | dcl_controller_timestamp | measurement_wavelength_beta | measurement_wavelength_cdom | measurement_wavelength_chl | raw_internal_temp | raw_signal_beta | raw_signal_cdom | raw_signal_chl | time_string
----------+-------+--------------+------------+-----------+-----------------------+------------+--------------------+-----------------------+---------------------+--------------+-------------+--------------------------+-----------------------------+-----------------------------+----------------------------+-------------------+-----------------+-----------------+----------------+-------------
 CE06ISSM | SBD17 | 06-FLORTD000 | 3626294400 | recovered | 3626296221.9110002518 |          1 |         3626296217 | 3626296221.9110002518 |      port_timestamp |           ok |        null |                     null |                        null |                        null |                       null |              null |               0 |               0 |              0 |        null
 CE06ISSM | SBD17 | 06-FLORTD000 | 3626294400 | recovered | 3626299829.3260002136 |          1 |         3626299824 | 3626299829.3260002136 |      port_timestamp |           ok |        null |                     null |                        null |                        null |                       null |              null |            1926 |              88 |            124 |        null
 CE06ISSM | SBD17 | 06-FLORTD000 | 3626294400 | recovered | 3626303422.8099999428 |          1 |         3626303418 | 3626303422.8099999428 |      port_timestamp |           ok |        null |                     null |                        null |                        null |                       null |              null |               0 |               0 |              0 |        null
 CE06ISSM | SBD17 | 06-FLORTD000 | 3626294400 | recovered | 3626307029.9180002213 |          1 |         3626307025 | 3626307029.9180002213 |      port_timestamp |           ok |        null |                     null |                        null |                        null |                       null |              null |            1902 |              87 |            125 |        null
 CE06ISSM | SBD17 | 06-FLORTD000 | 3626294400 | recovered | 3626310621.2720003128 |          1 |         3626310616 | 3626310621.2720003128 |      port_timestamp |           ok |        null |                     null |                        null |                        null |                       null |              null |               0 |               0 |              0 |        null
 CE06ISSM | SBD17 | 06-FLORTD000 | 3626294400 | recovered | 3626314229.4530000687 |          1 |         3626314224 | 3626314229.4530000687 |      port_timestamp |           ok |        null |                     null |                        null |                        null |                       null |              null |            1782 |              86 |            125 |        null
 CE06ISSM | SBD17 | 06-FLORTD000 | 3626294400 | recovered | 3626317823.0570001602 |          1 |         3626317818 | 3626317823.0570001602 |      port_timestamp |           ok |        null |                     null |                        null |                        null |                       null |              null |               0 |               0 |              0 |        null
 CE06ISSM | SBD17 | 06-FLORTD000 | 3626294400 | recovered | 3626321423.9200000763 |          1 |         3626321419 | 3626321423.9200000763 |      port_timestamp |           ok |        null |                     null |                        null |                        null |                       null |              null |               0 |               0 |              0 |        null
 CE06ISSM | SBD17 | 06-FLORTD000 | 3626294400 | recovered | 3626325023.4759998322 |          1 |         3626325018 | 3626325023.4759998322 |      port_timestamp |           ok |        null |                     null |                        null |                        null |                       null |              null |               0 |               0 |              0 |        null
 CE06ISSM | SBD17 | 06-FLORTD000 | 3626294400 | recovered |  3626328622.263999939 |          1 |         3626328617 |  3626328622.263999939 |      port_timestamp |           ok |        null |                     null |                        null |                        null |                       null |              null |               0 |               0 |              0 |        null
 CE06ISSM | SBD17 | 06-FLORTD000 | 3626294400 | recovered | 3626332229.4969997406 |          1 |         3626332224 | 3626332229.4969997406 |      port_timestamp |           ok |        null |                     null |                        null |                        null |                       null |              null |            1036 |              86 |            126 |        null
 CE06ISSM | SBD17 | 06-FLORTD000 | 3626294400 | recovered | 3626335827.5939998627 |          1 |         3626335822 | 3626335827.5939998627 |      port_timestamp |           ok |        null |                     null |                        null |                        null |                       null |              null |            1273 |              89 |            121 |        null
 CE06ISSM | SBD17 | 06-FLORTD000 | 3626294400 | recovered | 3626339422.0739998817 |          1 |         3626339417 | 3626339422.0739998817 |      port_timestamp |           ok |        null |                     null |                        null |                        null |                       null |              null |               0 |               0 |              0 |        null
 CE06ISSM | SBD17 | 06-FLORTD000 | 3626294400 | recovered | 3626343022.4409999847 |          1 |         3626343017 | 3626343022.4409999847 |      port_timestamp |           ok |        null |                     null |                        null |                        null |                       null |              null |               0 |               0 |              0 |        null
 CE06ISSM | SBD17 | 06-FLORTD000 | 3626294400 | recovered | 3626346622.2620000839 |          1 |         3626346617 | 3626346622.2620000839 |      port_timestamp |           ok |        null |                     null |                        null |                        null |                       null |              null |               0 |               0 |              0 |        null
 CE06ISSM | SBD17 | 06-FLORTD000 | 3626294400 | recovered |  3626350222.486000061 |          1 |         3626350217 |  3626350222.486000061 |      port_timestamp |           ok |        null |                     null |                        null |                        null |                       null |              null |               0 |               0 |              0 |        null
 CE06ISSM | SBD17 | 06-FLORTD000 | 3626294400 | recovered | 3626353825.9390001297 |          1 |         3626353821 | 3626353825.9390001297 |      port_timestamp |           ok |        null |                     null |                        null |                        null |                       null |              null |             977 |              85 |            125 |        null
 CE06ISSM | SBD17 | 06-FLORTD000 | 3626294400 | recovered | 3626357424.1789999008 |          1 |         3626357419 | 3626357424.1789999008 |      port_timestamp |           ok |        null |                     null |                        null |                        null |                       null |              null |               0 |               0 |              0 |        null
 CE06ISSM | SBD17 | 06-FLORTD000 | 3626294400 | recovered | 3626361024.2080001831 |          1 |         3626361019 | 3626361024.2080001831 |      port_timestamp |           ok |        null |                     null |                        null |                        null |                       null |              null |               0 |               0 |              0 |        null
 CE06ISSM | SBD17 | 06-FLORTD000 | 3626294400 | recovered |  3626364622.513999939 |          1 |         3626364617 |  3626364622.513999939 |      port_timestamp |           ok |        null |                     null |                        null |                        null |                       null |              null |               0 |               0 |              0 |        null
 CE06ISSM | SBD17 | 06-FLORTD000 | 3626294400 | recovered |  3626368223.388999939 |          1 |         3626368218 |  3626368223.388999939 |      port_timestamp |           ok |        null |                     null |                        null |                        null |                       null |              null |               0 |               0 |              0 |        null
 CE06ISSM | SBD17 | 06-FLORTD000 | 3626294400 | recovered | 3626371821.7150001526 |          1 |         3626371817 | 3626371821.7150001526 |      port_timestamp |           ok |        null |                     null |                        null |                        null |                       null |              null |               0 |               0 |              0 |        null
 CE06ISSM | SBD17 | 06-FLORTD000 | 3626294400 | recovered | 3626375423.8800001144 |          1 |         3626375419 | 3626375423.8800001144 |      port_timestamp |           ok |        null |                     null |                        null |                        null |                       null |              null |               0 |               0 |              0 |        null
 CE06ISSM | SBD17 | 06-FLORTD000 | 3626294400 | recovered | 3626379022.1669998169 |          1 |         3626379017 | 3626379022.1669998169 |      port_timestamp |           ok |        null |                     null |                        null |                        null |                       null |              null |               0 |               0 |              0 |        null

(24 rows)
